package.json

```diff
@@ -37,23 +38,22 @@
   "module": "dist/node/index.js",
   "types": "dist/node/index.d.ts",
   "scripts": {
-    "build": "node --experimental-loader esbuild-node-loader scripts/build.ts",
-    "test": "node --experimental-loader esbuild-node-loader --experimental-specifier-resolution=node ./node_modules/.bin/uvu tests",
+    "build": "node --experimental-loader esbuild-node-loader scripts/build.ts && tsc -p tsconfig.build.json",
+    "test": "vitest --globals",
```

Comment:
The build process is mostly the same, but given that vite doesn't really do anything with typegen I added an extra ts build config to handle that. I'll explain why an extra ts file was needed later. 

The `--global` flag to vitest is just so that we don't have to import `test`